zaldis/NeovimConfig
# Some setup.sh improvements

### Updates

- Spinner for loading steps
- Utils module with auxiliary functions
- Separated module to check Ubuntu dependencies
- Separated module to check MacOS dependencies 

### How to test

1. Run `scripts/setup.sh` script and follow it.
2. Run NeoVim and check that editor works as expected.

### Expected result

![image](https://user-images.githubusercontent.com/38065313/207234503-ab2b90c0-07c8-499a-8c7f-b0632c2b6eaa.png)
